polkadot/xcm/docs/src/cookbook/other_parachain_tokens/mod.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! # Supporting other parachain tokens
+//!
+//! This example shows how to configure a parachain to be able to handle other parachain tokens.
+//!
+//! The most important configuration is the `AssetTransactor`.
```

Comment:
Where can I learn what is an asset transactor in XCM? can it be backlinked here?

Reply:
`AssetTransactor` is an associated type in `xcm_executor::Config`. Don't think I can link to that, but I can link to the trait bound it has, `xcm_executor::traits::TransactAsset`